### PR TITLE
Add nest.js example

### PR DESCRIPTION
**Pull request by shicholas**

### CODE REVIEW

File: src/pages/postgraphile/usage-library.md [hunk 1]
```diff
@@ -218,6 +218,29 @@ if (middleware.options.watchPg) {
...
 ```
 
+For Nest, this might look something like:
+
+```js
+import { Controller, Get, Post, Req, Next, Res } from '@nestjs/common'
+import { Request, Response } from 'express'
+import { PostGraphileResponseNode } from 'postgraphile'
+import { middleware } from './postgraphile.middleware'
+import { PostGraphileResponseNest } from './postgraphileResponseNest'
```
Comment (benjie) [inline src/pages/postgraphile/usage-library.md:228]: Where is PostGraphileResponseNest defined?

**Review by benjie — CHANGES_REQUESTED** [review]
Thanks for your work on this; you seem to have referenced a file that was not included though?

Further, for consistency with the rest of the documentation please include semicolons in the code - `prettier` should do this for you (even in the markdown).

File: src/pages/postgraphile/usage-library.md (hunk 1 above)
Comment (shicholas) [reply src/pages/postgraphile/usage-library.md:228]: whoops that's not needed, will remove

**Review by benjie — APPROVED** [review]
Awesome; this looks really simple! :raised_hands:

**Pull request merged**